dev/TabView/TabViewItem.cpp

```diff
@@ -35,18 +40,39 @@ void TabViewItem::OnApplyTemplate()
                 winrt::AutomationProperties::SetName(closeButton, closeButtonName);
             }
 
+            // Setup the tooltip for the close button
+            auto tooltip = winrt::ToolTip();
+            auto closeButtonTooltipText = ResourceAccessor::GetLocalizedStringResource(SR_TabViewCloseButtonTooltip);
+            if (internalTabView != nullptr)
+            {
+                auto keyboardAcceleratorText = internalTabView->GetCloseButtonLocalizedKeyboardAccelerator();
+                if (keyboardAcceleratorText != hstring{})
+                {
+                    // TODO:
+                    // - Appending the tooltip content here...is there a better way to do this in C++/WinRT?
+                    // - I think we might also have to localize the parantheses? I.e. Asian languages uses different parantheses character
+                    //   than latin-based languages.
+                    // - This value is the same for all tabs in a tabview. Can we compute this only once and then use it for all tab items?
```

Comment:
Since we currently do not allow changing the keyboard accelerator, the complete localized string should be in the ressource file. 

If the close tab hotkey can be changed at some point, we won't be able to provide a localization anyway, so this would be something the developers would have to do then.

Reply:
Discussing this with @chingucoding led to the discovery of keyboard accelerator localization issue https://github.com/microsoft/microsoft-ui-xaml/issues/2023. Decided to wait for the team to decide if the current hard-coded value is ok for now or if localized values should be provided.

Localized values would have to be provided by the team as I don't know all the possible localizations for "Ctrl".

Reply:
We definitely cannot be concatenating localized strings. If the desired tooltip is "Close Tab (Ctrl+F4)" then that is exactly what should be in the TabViewCloseButtonTooltip resource. You can revert your changes for everything except en-us, localization will take care of it. :)

Reply:
@teaP Since no keyboard accelerators are available pre-RS3, should I add two string to the en-us resource file then (one including the accelerator, the other one without it)? I.e.:
![image](https://user-images.githubusercontent.com/1398851/75383234-47ddf780-58dc-11ea-836d-75eceb3af192.png)
I would then assign the correct string based on TabView.cpp's keyboard accelerator logic to `m_tabItemCloseKeyboardAccelerator` which then can be grabbed by a TabViewItem (as done in currently in my commits).

Reply:
How about just using TabViewCloseButtonName if there's no accelerator?

Reply:
TabViewCloseButtonName maps to "Close **T**ab" right now (capital T) whereas the suggestion for now is to use "Close tab" as chosen in Edge. I would also be fine using the upper-case variant though ("Close Tab") in which case we can reuse `TabViewCloseButtonName`.

Reply:
Oh, got it. Well, it's fine to have both strings then, in my opinion.

Reply:
@teaP Locally, I now have the following logic in TabViteItem.cpp:
```cs
if (internalTabView)
{
    // Setup the tooltip for the close button
    auto tooltip = winrt::ToolTip();
    tooltip.Content(box_value(internalTabView->GetTabCloseButtonTooltipText()));
    winrt::ToolTipService::SetToolTip(closeButton, tooltip);
}
```
This means the tab close button will only get the tooltip content "Close tab (...)" when the tab item is part of a TabView control. Otherwise it will display the current content (which is the tab item header). 
I checked the TabView code and from the looks ok it, if a TabViewItem does not belogn to a TabView, it cannot be closed anyway (i.e. no [TabViewItem.CloseRequested](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.tabviewitem.closerequested?view=winui-2.2) event is raised). Is displaying the tab item header fine then (no explicit tooltip for a tab's close button), or should we also create a custom tooltip reading "Close tab" in this case?

Reply:
It's not supported for the TabViewItem to be outside a TabView, so you don't need to handle that case.

Reply:
Pushed changes now, sorry for the delay. I had to rebuilt my entire WinUI built system and also got blue screens when building. I managed to get the build system working again for me...for now...